### PR TITLE
Coveralls

### LINKED ISSUE #218: Integrate with coveralls
Integrate the project with [coveralls](https://coveralls.io/) to add a fancy badge in the readme, like this one from lodash:

[Coverage Status](https://coveralls.io/github/lodash/lodash?branch=master)

And the setup is pretty simple:

1. Create an account with your github at the website.
2. Add this repo on the website:

screenshot coveralls

3.  Add a simple script to the `package.json`:
```js
"coveralls": "npm run coverage && nyc report --reporter=text-lcov | coveralls"
```
4. And run this script with [travis](https://travis-ci.org/), just adding this snippet below to the end of `travis.yml`
```yml
after_script:
  - npm i -g coveralls
  - npm run coveralls
```

### PR DESCRIPTION
**Pull request by thiamsantos**
Integrate with [coveralls](https://coveralls.io/), closes #218

### CODE REVIEW

**Comment by thiamsantos** [comment]
Everything seems to work properly :tada:

File: .travis.yml [hunk 1]
```diff
@@ -7,4 +7,7 @@ notifications:
       - https://webhooks.gitter.im/e/c75729ddc58d8813cc6d
     on_success: always
     on_failure: always
-    on_start: never
\ No newline at end of file
+    on_start: never
+after_script:
+  - npm i -g coveralls
```
Comment (xymostech) [inline .travis.yml]: Can you just put this as a `devDependency` in package.json?

**Comment by thiamsantos** [comment]
[clabot:check]

File: .travis.yml (hunk 1 above)
Comment (thiamsantos) [inline .travis.yml]: Done.

**Comment by xymostech** [comment]
Awesome. Is it going to comment in the thread every time there's a new build? We already fail tests if the coverage is <100%, so we probably don't need that... maybe I can go turn it off.

**Pull request merged**